CMakeLists.txt

```diff
@@ -309,13 +309,13 @@ if(WIN32 OR CMAKE_USE_DARWINSSL OR CMAKE_USE_WINSSL OR CMAKE_USE_MBEDTLS)
 endif()
 option(CMAKE_USE_OPENSSL "Use OpenSSL code. Experimental" ${openssl_default})
 
-collect_true(enabled_ssl_options enabled_ssl_options_count
+count_true(num
```

Comment:
Given the scope of the variable, calling it "num" is probably not such a good idea. I'd rather leave the old name here.

Reply:
Fixed